### PR TITLE
添加 x-img blazy container 配置项

### PR DESCRIPTION
**Pull request by cx-emerge**
Please makes sure the items are checked before submitting your PR, thank you!

* [ ] `Rebase` before creating a PR to keep commit history clear.
* [ ] `Only One commit`
* [ ] No `eslint` errors

添加 x-img blazy container 配置项

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/airyland/vux/pull/458" target="_blank">ReviewNinja

File: src/components/x-img/index.vue [hunk 1]
```diff
@@ -21,7 +21,7 @@ export default {
     this.$el.setAttribute('id', id)
     this.$el.setAttribute('data-src', this.src)
     this.blazy = new Blazy({
-      scroller: this.scroller,
```
Comment (airyland) [inline src/components/x-img/index.vue]: scroller 不能去掉，这是为了让x-img在scroller里能正常使用添加的一个scroller引用。

File: src/components/x-img/index.vue [hunk 2]
```diff
@@ -48,7 +49,11 @@ export default {
       defaut: 100
     },
     class: String,
-    scroller: Object
+    scroller: Object,
+    container: {
+      type: String,
+      default: 'window'
```
Comment (airyland) [inline src/components/x-img/index.vue]: window?

Comment (cx-emerge) [inline src/components/x-img/index.vue]: [blazy blog](http://dinbror.dk/blog/blazy/?ref=github)

```
container [string]  
(window)
```

Comment (airyland) [inline src/components/x-img/index.vue]: There is no selector named `'window'`, there is only one `window`. Blazy will use `window` if no container is specified.
https://github.com/dinbror/blazy/blob/master/blazy.js#L54

**Comment by airyland** [comment]
Thanks. Could you squash the commits into one ?
http://www.cnblogs.com/dsxniubility/p/4460834.html

**Pull request closed without merging**